documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -12,56 +12,854 @@ SPDX-License-Identifier: MPL-2.0
+
+Go back to the <<markdown-where-needed>> overview.
+
+[[sources-markdown]]
+=== Acknowledge sources (Markdown)
+
+To indicate the cited sources used within the `description` attribute, use the Markdown blockquote syntax.
+
+TIP: Use *SOURCE:* as a prefix. +
+(While not required by Markdown, this practice helps distinguish notes, examples, and sources in Aspect Models.)
+
+*When to use?* A "source" acknowledges a source that was cited in the Description.
+
+NOTE: To generally link to related resources for further reference, use xref:ROOT:modeling-guidelines.adoc#attributes-that-all-model-elements-have[the `samm:see` attribute].
+
+.Example: Markdown blockquote syntax for indicating a cited source
+[source,turtle,subs="attributes+,+quotes"]
+----
+samm:description """
+   Lorem ipsum description text, citing a source.
+   > SOURCE: Indicate source, optionally add [a link](https://www.example.com)
+   """@en ;
+----
+
+https://spec.commonmark.org/dingus/?text=%20%20%20Lorem%20ipsum%20description%20text%2C%20citing%20a%20source.%0A%20%20%20%3E%20SOURCE%3A%20Indicate%20source%2C%20optionally%20add%20%5Ba%20link%5D(https%3A%2F%2Fwww.example.com)%0A[Try it^]
+
+.Example descriptions acknowledging cited sources
+[source,turtle,subs="attributes+,+quotes"]
+----
+<...>
+   samm:preferredName "digital representation"@en ;
+   samm:description """
+      Information and services representing an entity from a given viewpoint.
+      > SOURCE: IEC 63278-1:2023, editorial changes, no examples
+      """@en ;
+   <...>
+
+<...>
+   samm:preferredName "service"@en ;
+   samm:description """
+      Distinct part of the functionality that is provided by an entity through interfaces.
+      > SOURCE: IEC 63278-1:2023; IEC 60050-741:2020, 741-01-28
+      """@en ;
+   <...>
+
+<...>
+   samm:preferredName "system"@en ;
+   samm:description """
+      Interacting, interrelated, or interdependent elements forming a complex whole.
+      > SOURCE: IEC 63278-1:2023; IEC TS 62443-1-1:2009, 3.2.123
+      """@en ;
+   <...>
+----
+
+https://spec.commonmark.org/dingus/?text=%23%20Digital%20representation%0A%20%20%20Information%20and%20services%20representing%20an%20entity%20from%20a%20given%20viewpoint.%0A%20%20%20%3E%20SOURCE%3A%20IEC%2063278-1%3A2023%2C%20editorial%20changes%2C%20no%20examples%0A%0A%23%20Service%0A%20%20%20Distinct%20part%20of%20the%20functionality%20that%20is%20provided%20by%20an%20entity%20through%20interfaces.%0A%20%20%20%3E%20SOURCE%3A%20IEC%2063278-1%3A2023%3B%20IEC%2060050-741%3A2020%2C%20741-01-28%0A%0A%23%20System%0A%20%20%20Interacting%2C%20interrelated%2C%20or%20interdependent%20elements%20forming%20a%20complex%20whole.%0A%20%20%20%3E%20SOURCE%3A%20IEC%2063278-1%3A2023%3B%20IEC%20TS%2062443-1-1%3A2009%2C%203.2.123%0A[Try it^] – This roughly sketches how the Preferred Name could be parsed and combined with the Description within an output.
```

Comment:
The formatting looks a little bit strange: font of SOURCE is completely different from text and much bigger: is this really like this?